code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1147,6 +1151,7 @@
 
 	icon_selected = 1
 	icon_selection_tries = 0
+	sprite_type = " [robot_species]"

```

Comment:
I'd suggest not putting a space in the sprite_type, but instead up where name as a whole is formed.

Reply:
If I do that then drones and people who don't pick a sprite will have double spaces in the name. u wu

Reply:
Something like '[sprite_type ? "[sprite_type] " : ""]' maybe?